addons/initialise-sprite-position/addon.json

```diff
@@ -37,6 +37,26 @@
       "name": "Randomize the position of library sprites",
       "type": "boolean",
       "default": false
+    },
+    {
+      "id": "duplicate",
+      "name": "Behaviour when duplicating sprites",
+      "type": "select",
+      "default": "keep",
+      "potentialValues": [
+        {
+          "id": "custom",
+          "name": "Send to custom x/y values"
```

Comment:
When I chose this option while testing, I expected two new settings to pop up. (using `setting.if`). Of course, then I noticed this referred to the same x/y values as the default for new sprites.
Maybe it could be a bit more clear that it's referring to the x/y values. `specified` instead of `custom`?

Reply:
Soumds good to me 👍